### PR TITLE
feat: Review the display of Reel video - MEED-6886 - Meeds-io/MIPs#124

### PR DESCRIPTION
**Pull request by SaraBoutej**
Prior to this change, embedded reel videos were not displayed correctly due to styles added to the video. 
This PR adjusts the CSS styles for reel videos to ensure proper display

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityEmbeddedHTML.vue [hunk 1]
```diff
@@ -41,10 +41,12 @@ export default {
   data: () => ({
     maxWidth: 320,
     elementReady: false,
+    isReelVideo: false
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityEmbeddedHTML.vue]:
```suggestion
    isVideo: false
```

Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityEmbeddedHTML.vue]: removed

File: webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityEmbeddedHTML.vue [hunk 2]
```diff
@@ -44,7 +44,8 @@ export default {
   }),
   computed: {
     embeddedHTML() {
-      return this.activity && this.activity.templateParams && this.activity.templateParams.html;
+      const htmlElement = this.activity && this.activity.templateParams && this.activity.templateParams.html;
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityEmbeddedHTML.vue]:
```suggestion
      const htmlElement = this.activity?.templateParams?.html;
```

Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/content/ActivityEmbeddedHTML.vue]: fixed

**Review by boubaker — APPROVED** [review]

**Pull request merged**